apps/www/components/ActionBar/ShareButtons.js

```diff
@@ -46,33 +46,6 @@ const ShareButtons = ({
     }
   }, [copyLinkSuffix])
 
```

Comment:
The only place this was ever used was the end of a series overview. I removed that so the ShareButtons component now only ever renders the sharebuttons without a native app fallback.